### ISSUE TITLE
On comments section, wrong username is used

### ISSUE THREAD

**Issue by scrahe** [coopeeo/customname-geodemod#3]
If you click on your comments, it still says your username.

**Comment by scrahe** [comment]
Yes, if you click the button to show your comments on your profile, it shows the wrong name.

**Comment by scrahe** [comment]
To specify, the title is incorrect.

**Comment by coopeeo** [comment]
yea when profile uses commentcell, its different structure from level comments

**Comment by coopeeo** [comment]
Hey @scrahe, may you try latest release on the index

**Comment by scrahe** [comment]
Part of it was fixed, but the title is still wrong.
![dummy](https://github.com/coopeeo/customname-geodemod/assets/55860416/410c1dbe-3e8e-4389-a9f5-15807721aa06)

**Comment by coopeeo** [comment]
ooooooooooooh that

**Comment by scrahe** [comment]
It works now.
